source/aggregation-tutorials/group-total.txt

```diff
@@ -0,0 +1,225 @@
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+This aggregation tutorial demonstrates how to group and analyze data
+describing customer orders. The aggregated result is a list of customers
+who purchased items in 2020. The results include information about
+each customer's order history.
+
+This example uses one collection, ``orders``, which contains documents
+describing individual orders for products in a shop.
+
+An order can only correspond to one customer, so the documents are
+grouped by the ``customer_id`` field, which provides the email address
+of the customer.
+
+Before You Get Started
+----------------------
+
+Before you start following this tutorial, follow the
+instructions in the :ref:`node-agg-tutorial-template-app` section of the
+Aggregation Tutorials page to set up the template app.
+
+Once you set up the app, access the ``orders`` collection by updating
+the collection name placeholder, as shown in the following code:
+
+.. literalinclude:: /includes/aggregation/group-total.js
+   :language: javascript
+   :copyable: true
+   :start-after: start-coll
+   :end-before: end-coll
+   :dedent:
+
+.. note::
+   
+   Since this tutorial accesses documents in only one collection, you
+   can delete any lines in the template app that reference a second
+   collection.
+
+Insert sample data into the ``orders`` collection by
+pasting the following sample documents into the ``sampleData1`` array:
+
+.. literalinclude:: /includes/aggregation/multi-field-join.js
+   :language: javascript
+   :copyable: true
+   :start-after: start-orders
+   :end-before: end-orders
+   :dedent:
+
+Tutorial
+--------
+
+.. procedure::
+   :style: connected
+
+   .. step:: Build The Aggregation Pipeline
+
+      .. procedure::
+         :style: connected
+
+         .. step::
+
+            First, add a :manual:`$match
+            </reference/operator/aggregation/match>` stage that matches
+            orders placed in 2020:
+
+            .. literalinclude:: /includes/aggregation/group-total.js
+               :language: javascript
+               :copyable: true
+               :start-after: start-match
+               :end-before: end-match
+               :dedent:
+
+         .. step::
+
+            Next, add a :manual:`$sort
+            </reference/operator/aggregation/sort>` stage to set an
+            ascending sort on the ``orderdate`` field, so that you can
+            find the first purchase data for each customer in the next
+            stage:
+
+            .. literalinclude:: /includes/aggregation/group-total.js
+               :language: javascript
+               :copyable: true
+               :start-after: start-sort1
+               :end-before: end-sort1
+               :dedent:
+
+         .. step::
+
+            Add a :manual:`$group
+            </reference/operator/aggregation/group>` stage to group
+            orders by the value of the ``customer_id`` field. In this
+            stage, calculate the following new fields:
```

Comment:
Suggestion:

I think "calculate" may not describe "the list of all the customer's purchases". 
I also think it would be good to describe the user's direct tasks when using the imperative mood. Since they need to create aggregation operations to produce the new data fields, perhaps this wording would work:

"Add aggregation operations to the stage that output the following fields in the result documents:"